Traceback (most recent call last):
  File "D:\MCTS-AHD-master\problems\tsp_constructive\eval.py", line 11, in <module>
    from gpt import select_next_node_v2 as select_next_node
  File "D:\MCTS-AHD-master\problems\tsp_constructive\gpt.py", line 1
    importance of immediate vs. lookahead distances while penalizing large angular deviations between current and destination paths.}
                                                                                                                                    ^
SyntaxError: unmatched '}'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\MCTS-AHD-master\problems\tsp_constructive\eval.py", line 13, in <module>
    from gpt import select_next_node
  File "D:\MCTS-AHD-master\problems\tsp_constructive\gpt.py", line 1
    importance of immediate vs. lookahead distances while penalizing large angular deviations between current and destination paths.}
                                                                                                                                    ^
SyntaxError: unmatched '}'
